test/controllers/state_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
Please remote this tests, and make sure we change the specs to `rspec`, you can make it on another branch though